### ISSUE TITLE
Support database connector to act when an action in complete

### ISSUE THREAD

**Issue by lecoueyl** [social-manager-tools/instagram-bot.js#70]
This is not an issue but more of a feature request / proposition
This bot is really great and I don't have encountered an issue with it yet, great job!

### Versions

Replace the values below with your own:

- **Bot Version:** v0.8.2
- **Node Version:** v10.9 (Bot require >= 7.6)
- **Operating System:** Ubuntu 14.04
- **Browser:** Google Chrome 64

### Expected Behavior

Please describe the program's expected behavior.

```
When a photo is liked, it would be nice to have the option to send this event to a database to keep track of the numbers of likes/follow the script accomplished, with the link.
Then with the data you can cross it to view how many followers you gain by day using the bot.

The only way I could figure it with the actual version is to crawl the output log and put this in a database
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
Console log  and events send to Slack are only supported
```

**Comment by ptkdev** [comment]
I add this feature in major release `v0.9.0` with for `follow/defollow mode` and `like mode`. 
Imho sql-lite database in `/databases/` folder is good solution, you prefer other type of db?

**Comment by lecoueyl** [comment]
Great! I think this is a good solution too.

**Comment by ptkdev** [comment]
Available in release **v0.9.0**

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
